### ISSUE TITLE
[Pspe] random client crashes

### ISSUE THREAD

**Issue by Wesley-VanNeck** [ProtocolSupport/ProtocolSupportPocketStuff#94]
If you guys wanth a logcat from when that happens I can provide it when am home

**Comment by 7kasper** [comment]
Yeah that might help. I think its a missing remap or something. Can you try isolating the issue? E.g. a to a certain chunk or block or entity...
